sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py

```diff
@@ -48,179 +36,99 @@
     WebNotificationHook,
 )
 
+# for pytest.parametrize
+subscription_key = os.getenv("METRICS_ADVISOR_SUBSCRIPTION_KEY", "metrics_advisor_subscription_key")
```

Comment:
In https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/metricsadvisor/tests.yml, we set 
AZURE_METRICS_ADVISOR_SUBSCRIPTION_KEY
but here we use 
METRICS_ADVISOR_SUBSCRIPTION_KEY

Is it by design?

Reply:
The tests.yml is updated in this PR to drop the AZURE_ prefix: https://github.com/Azure/azure-sdk-for-python/blob/1d3cb9c3934752376ee2064bb9988e16543c1dc5/sdk/metricsadvisor/tests.yml

Main motivations were to shorten the names and more closely match the naming convention we use for env vars with the PowershellPreparer work (yes I know we don't use PowershellPreparer in this library yet, but if we ever do it the renames will be done).

Reply:
Does this (shortening the env name) also apply to other libraries?

Reply:
If using the PowershellPreparer, the env vars must fit this convention for them to be properly sanitized so it does apply in that case. I don't have the context of why we decided to do it like that. If you want to keep AZURE_ for this library, I can revert those renames.

Reply:
I don't have a strong preference to have AZURE_ prefix or not. But I would like to keep consistency across our libraries then we can add it into our guideline.

One of my concerns is some env names may be too generic if we don't have the AZURE_ prefix. (Maybe that's the reason we added it?)

Reply:
I can't link to it exactly, but in our tests doc under 3) we do say to create the env vars with this naming convention: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#create-live-test-resources